app/controllers/api/v1/providers_controller.rb

```diff
@@ -6,7 +6,7 @@ def index
         paginate json: @providers
       end
 
-      def show
+      def show_by_recruitment_cycle
```

Comment:
I think `recruitment_cycle` is enough

Reply:
`show_by_recruitment_cycle` that can be removed